HISTORY.md

```diff
@@ -1,5 +1,17 @@
 # History
 
+## 0.1.3 - 2023-02-02
```

Comment:
Isn't this going to be a minor release, so 0.2.0?

Reply:
Good catch!